content/docs/gke/authentication.md

```diff
@@ -81,12 +81,12 @@ gcloud iam service-accounts add-iam-policy-binding \
 
 4. Manually create a profile for user1 and specify the GCP service account to bind in `plugins` field:
 
-```
+```yaml
 apiVersion: kubeflow.org/v1beta1
 kind: Profile
 metadata:
   name: profileName   # replace with the name of the profile (the user's namespace name)
-spec:
+  spec:
```

Comment:
The indentation is good now. Just one more fix: Remove the additional space added at the beginning of this line. (The `spec` line should align with `metadata` - the format given in the issue is incorrect in this respect.)

Reply:
@sarahmaddox - If the spec is aligned with the metadata, then, the owner property should also be aligned with the metadata or it will be inside spec?

Reply:
@sarahmaddox - Aligned spec with the metadata. Can you please check this again? Thank you.